### PR TITLE
Plot z

### PR DESCRIPTION
**Pull request by jennyfolkesson**
I've not added tests because testing a saved figure is a pain. I could add a general test to make sure that the function writes a file if you think that makes sense?

### CODE REVIEW

File: micro_dl/plotting/plot_utils.py [hunk 1]
```diff
@@ -74,6 +76,98 @@ def save_predicted_images(input_batch, target_batch, pred_batch,
         plt.close(fig)
 
 
+def plot_xyz(image_dir,
```
Comment (anithapk) [inline micro_dl/plotting/plot_utils.py]: plot_xyz sounds like a scatter plot in 3D, could we please rename this to: save_center_orthogonal_view
or save_mid_orthogonal_slice or the like?

File: micro_dl/plotting/plot_utils.py [hunk 2]
```diff
@@ -74,6 +76,98 @@ def save_predicted_images(input_batch, target_batch, pred_batch,
...
+             pos_idx,
+             fig_name,
+             mean_std=None,
+             tol=1,
+             font_size=15,
+             color_map='gray',
+             fig_title=None,
+             margin=20,
+             z_scale=5,
+             channel_str=None):
```
Comment (anithapk) [inline micro_dl/plotting/plot_utils.py]: Please feel free to ignore this, it seems like we can group these params into image related and figure related params, may be two dicts image_params: {pos_idx, mean_std, tol, z_scale, channel_str} and fig_params:{fig_name, font_size, color_map, fig_title, margin}. Also could we please replace tol with per_clip_limits as tol reminds me of tolerance or epsilon

File: micro_dl/plotting/plot_utils.py [hunk 3]
```diff
@@ -74,6 +76,98 @@ def save_predicted_images(input_batch, target_batch, pred_batch,
...
+    :param int z_scale: How much to upsample in z (to be able to see xz and yz)
+    :param str channel_str: If there's more than one channel in image_dir
+        (e.g. input image dir as opposed to predictions) use this str to select
+        which channel to build z-stack from
+    """
+
+    search_str = os.path.join(image_dir, "*p{:03d}*".format(pos_idx))
+    slice_names = natsort.natsorted(glob.glob(search_str))
+
+    if channel_str is not None:
```
Comment (anithapk) [inline micro_dl/plotting/plot_utils.py]: guess one could pass channel_ids as channel_str here? maybe specify what this str could be in doc string?

File: micro_dl/plotting/plot_utils.py (hunk 2 above)
Comment (jennyfolkesson) [reply micro_dl/plotting/plot_utils.py]: tol was copied from the plot function above, so I'll update that one too while I'm at it. :-)

Comment (anithapk) [reply micro_dl/plotting/plot_utils.py]: yes please 😸

**Pull request merged**